### DISCUSSION TITLE
Comparison to @capacitor-community/firebase-analytics

### DISCUSSION THREAD

**Discussion by panstromek** [capawesome-team/capacitor-firebase#478] [Q&A] [answered]
Hi, are there some notable differences between `@capacitor-firebase/analytics` and the older `@capacitor-community/firebase-analytics` plugin? I'm trying to decide what to use and how those options behave, but I can't seem to find any comparison.

Looks like `capacitor-firebase` is built on newer version of web SDK and supports some more methods on the web (`setScreenName`). Is there something else that's good to know?

**Comment by robingenz** [comment, marked as answer]
@capacitor-firebase/analytics is more up-to-date. You also won't get any SDK version conflicts if you decide to use other Firebase SDKs as well because all Firebase plugins in this repository are compatible.
